### PR TITLE
10.0 basic image export

### PR DESCRIPTION
**Pull request by sebastienbeau**
include https://github.com/akretion/odoo-shopinvader/pull/99

### CODE REVIEW

File: connector_locomotivecms/models/keychain.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
```
Comment (bealdav) [inline connector_locomotivecms/models/keychain.py]: openerp ?

Comment (sebastienbeau) [reply connector_locomotivecms/models/keychain.py]: OUPS !

Comment (bealdav) [reply connector_locomotivecms/models/keychain.py]: g lé djà vu ailleurs ds tes PR recentes. les 2 marches encore

File: shopinvader/models/shopinvader_image_mixin.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ShopinvaderImageMixin(models.AbstractModel):
+    _name = 'shopinvader.image.mixin'
+
+    images = fields.Serialized(
```
Comment (lmignon) [inline shopinvader/models/shopinvader_image_mixin.py:12]: @sebastienbeau `images`is not really a list of images. It's a ist of images informations. I'm not convinced by the name... (a details)

**Review by lmignon — APPROVED** [review]

**Pull request merged**